src/Shared/Recruit.Vacancies.Client/Infrastructure/QueryStore/Projections/LiveVacancy/TrainingProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections.LiveVacancy
+{
+    public class TrainingProvider
+    {
+        public long Ukprn { get; set; }
+        public string Name { get; set; }
+        public Address Address { get; set; }

```

Comment:
The one you are mapping to doesn't map Address so you could probably leave out the property on this class.